workspaces/orchestrator/plugins/orchestrator/src/components/WorkflowInstancePageContent.tsx

```diff
@@ -113,14 +125,62 @@ export const WorkflowInstancePageContent: React.FC<{
     return res.data;
   }, [orchestratorApi, workflowId]);
 
+  const [isVariablesDialogOpen, setIsVariablesDialogOpen] = useState(false);
+
+  const toggleVariablesDialog = React.useCallback(() => {
+    setIsVariablesDialogOpen(prev => !prev);
+  }, []);
+
+  const VariablesDialogContent = () => (
```

Comment:
Nit: by moving this component out of this function and passing `instanceVariables` as its property, the code will be cleaner and the component will not be re-created with every run.

Anyway, not a big issue in that particular case.